usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/DateTime.java

```diff
@@ -55,6 +58,44 @@ public DateTime(int year, int month, int day, int hour, int minutes, int seconds
         }
     }
 
+    public DateTime(String dateTime) {
+        String[] dateTimeParts = dateTime.split(DATE_TIME_SEPARATOR);
+        readDate(dateTimeParts[DATE]);
+        readTime(dateTimeParts[TIME]);
+    }
+
+    public DateTime(String dateTime, boolean isFull) {
+        if (isFull) {
+            String[] dateTimeParts = dateTime.split(DATE_TIME_SEPARATOR);
+            readDate(dateTimeParts[DATE]);
+            readTime(dateTimeParts[TIME]);
+        } else {
+            readDate(dateTime);
+            this.hour = this.minutes = this.seconds = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inner assignments should be avoided.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5351293)